features/step_definitions/parameter_store_steps.rb

```diff
@@ -0,0 +1,15 @@
+Given(/^(?:a|the) SSM parameter(?: named)? "([^"]*)" with value "([^"]*)" in region "([^"]*)"$/) do |parameter_name, parameter_value, parameter_region|
+  File.open("/tmp/humpy-log", 'a') { |file| file.write("In Here name: #{parameter_name} value: #{parameter_value} \n") }
```

Comment:
I'm assuming this was in here for debugging? Can it be removed?

Reply:
Yep, has been 17501ef  Was debugging why everything appeared to be stubbed, before I discovered `lib/stack_master/testing.rb`